src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -198,16 +198,16 @@ public static void ExtractAll(ZipArchive zip, string destination, Action<int, in
 			int i = 0;
 			int total = zip.Entries.Count;
 			foreach (var entry in zip.Entries) {
-
-				if (string.Equals(entry.FullName, "__MACOSX", StringComparison.OrdinalIgnoreCase) ||
-						string.Equals(entry.FullName, ".DS_Store", StringComparison.OrdinalIgnoreCase))
+				if (entry.FullName.Contains ("__MACOSX") ||
```

Comment:
On the one hand, I'm wrong in saying that `.Contains()` is locale-sensitive. [It isn't](https://msdn.microsoft.com/en-us/library/dy85x1sa(v=vs.110).aspx):

> This method performs an ordinal (case-sensitive and culture-insensitive) comparison

Alas, that means it's case sensitive. Hopefully that won't break anything.

That said, it'll accept with `some_file_name_that_contains_MACOSX.class`, so if you happen to have some resource that you *want* that *contains* `_MACOSX`, you're screwed.

I would thus suggest *two* checks for `_MACOSX`, for a total of three:

    if (entry.FullName.Contains ("/__MACOSX/") ||
            entry.FullName.EndsWith ("/__MACOSX", StringComparison.OrdinalIgnoreCase)) ||
            entry.FullName.EndsWith ("/.DS_Store", StringComparison.OrdinalIgnoreCase)))
        continue;

Additionally, the commit message should be updated to use two `_`s to be consistent with the code.